android/app/src/main/java/com/expensify/chat/customairshipextender/CustomNotificationProvider.java

```diff
@@ -106,6 +109,9 @@ protected NotificationCompat.Builder onExtendBuilder(@NonNull Context context, @
             builder.setPriority(PRIORITY_MAX);
         }
 
+        // Set sound for versions below Oreo
+        builder.setSound(getSoundFile(context));
```

Comment:
Could we wrap this in a requires SDK condition like [this one](https://github.com/Expensify/App/blob/eb6e6849dd0e967af91aebc2fc6b5d1d023b9452/android/app/src/main/java/com/expensify/chat/customairshipextender/CustomNotificationProvider.java#L105-L110)? Just in case

Reply:
And update your comment to saywhere it gets set for Oreo and above